internal/provider/data_source_certificate_test.go

```diff
@@ -208,7 +208,7 @@ func TestAccDataSourceCertificate_BadSSL(t *testing.T) {
 					r.TestCheckResourceAttr("data.tls_certificate.test", "certificates.1.signature_algorithm", "SHA256-RSA"),
 					r.TestCheckResourceAttr("data.tls_certificate.test", "certificates.1.public_key_algorithm", "RSA"),
 					r.TestCheckResourceAttr("data.tls_certificate.test", "certificates.1.is_ca", "false"),
-					r.TestCheckResourceAttr("data.tls_certificate.test", "certificates.1.sha1_fingerprint", "a33e459e760bff66228f99ab9c3c4b0259afaf6c"),
+					r.TestCheckResourceAttr("data.tls_certificate.test", "certificates.1.sha1_fingerprint", "6922cd864f3c6299f6e751a019e5ddcdbc415a71"),
```

Comment:
I should have done this when I did the same update last year in #348, but can you add a comment describing that this value is expected to change? (for future maintainers)

https://github.com/hashicorp/terraform-provider-tls/pull/348

Reply:
Good shout. Have added a maintainer note.